lib/logstash/inputs/kafka.rb

```diff
@@ -410,6 +416,7 @@ def create_consumer(client_id, group_instance_id)
 
       props.put(kafka::AUTO_COMMIT_INTERVAL_MS_CONFIG, auto_commit_interval_ms.to_s) unless auto_commit_interval_ms.nil?
       props.put(kafka::AUTO_OFFSET_RESET_CONFIG, auto_offset_reset) unless auto_offset_reset.nil?
+      props.put(kafka::AUTO_CREATE_TOPICS, auto_create_topics) unless auto_create_topics.nil?
```

Comment:
Given the setting is called [allow.auto.create.topics](https://kafka.apache.org/documentation/#consumerconfigs_allow.auto.create.topics):

```suggestion
      props.put(kafka::ALLOW_AUTO_CREATE_TOPICS, auto_create_topics) unless auto_create_topics.nil?
```